packages/directlinespeech/src/createAdapters.js

```diff
@@ -92,20 +92,17 @@ export default async function create({
 
   // Supported options can be found in DialogConnectorFactory.js.
 
+  // Setting the language use for recognition.
   config.setProperty(PropertyId.SpeechServiceConnection_RecoLanguage, speechRecognitionLanguage);
 
-  // The following code set the output format. But currently, none of the following works for setting detailed output format.
-  // We will leave these code commented until the Speech SDK support, possibly it in one of the way mentioned below.
+  // The following code set the output format.
+  // As advised by Speech team, this API may change in the future.
```

Comment:
Is there documentation for this? If yes, can we link?

Reply:
No doc.